src/borg/helpers/manifest.py

```diff
@@ -99,13 +102,13 @@ def list(self, *, glob=None, match_end=r'\Z', sort_by=(), first=None, last=None,
 
     def list_considering(self, args):
         """
-        get a list of archives, considering --first/last/prefix/glob-archives/sort cmdline args
+        get a list of archives, considering --first/last/prefix/glob-archives/sort/consider-checkpoints cmdline args
         """
         if args.location.archive:
-            raise Error('The options --first, --last, --prefix and --glob-archives can only be used on repository targets.')
+            raise Error('The options --first, --last, --prefix, --glob-archives, and --consider-checkpoints can only be used on repository targets.')
```

Comment:
is ", and" grammatically correct?

Reply:
It's an Oxford comma (https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/). But I'm happy to remove it if it's not preferred.

Reply:
ok, as long as it is not wrong, we can keep it.